packages/form/src/index.ts

```diff
@@ -10,6 +10,6 @@ export function configure(config: FrameworkConfiguration) {
   config.globalResources([
     PLATFORM.moduleName('./ux-form'),
     PLATFORM.moduleName('./ux-field'),
-    UxSubmitCustomAttribute
+    PLATFORM.moduleName('./ux-submit-attribute'),
```

Comment:
we need either this or `@customAttribute..` only. I think the decorator should be preferred, as it can help with optimization

Reply:
So what you mean is that we should have one OR the other but not BOTH ?

Reply:
Yes, having one is enough. Having 2 is the same. Just the downside of `PLATFORM.moduleName` is it disable some optimization of webpack on the module referred

Reply:
What about `@customElement` ? Should we also not use `PLATFORM.moduleName()` when there is an explicit decorator on the custom element ?

Reply:
no, we don't have to, but it's good to be explicit. `PLATFORM.moduleName` does a few things:
- deoptimize a module, to keep the names of its exported members kept as is
- signal webpack to register a module as dependency of current module

Reply:
Yeah, I've learnt with the Aurelia team that we aim to be as explicit as possible. This is great.

The most explicit is the usage of `@customAttribute()` and `@customElement()` right ?

So, for the `@customAttribute` you say that we _should not_ use PLATFORM.moduleName().

Just want to clarify to make sure I get it right for the future... Regarding `@customElement`. _Should we use_ PLATFORM.moduleName() or _should we not_ (and benefit from better module optimization) ?

Reply:
For custom element. ModuleName is needed because we need to inform webpack that it needs to bundle the corresponding view too, beside "do not rename export" signal